CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+## Adding a new rule
+New rules can be implemented in two ways:
+- as a `gosec.Rule` -- these define an arbitrary function which will be called on every AST node in the analyzed file, and are appropriate for rules that mostly need to reason about a single statement.
+- as an Analyzer -- these can operate on the entire program, and receive an [SSA](https://pkg.go.dev/golang.org/x/tools/go/ssa) representation of the program. This type of rule is useful when you need to perform a more complex analysis that requires a great deal of program context. 
```

Comment:
I think is a `representation` of a package. The current SSA representation doesn't support at call graph of a packages.

Reply:
I think I've fixed this-- does the new text look better?